imageroot/actions/configure-module/10validate_timezone

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+import json
+import sys
+import os
+import agent
+import subprocess
+
+# Try to parse the stdin as JSON.
+# If parsing fails, output everything to stderr
+data = json.load(sys.stdin)
+
+agent_id = os.getenv("AGENT_ID", "")
+if not agent_id:
+    raise Exception("AGENT_ID not found inside the environemnt")
+
+
+if data.get("timezone") is not None and data["timezone"] != os.environ["WEBTOP_TIMEZONE"]:
```

Comment:
:bulb: Using `in` operator, the Python way to check if a key is in the dictionary: 


```suggestion
if "timezone" in data and data["timezone"] != os.environ["WEBTOP_TIMEZONE"]:
```